### PR TITLE
JSAB2-79 Fix common styles

### PR DESCRIPTION
**Pull request by Kyya**

### CODE REVIEW

**Comment by Kyya** [comment]
subCardView done ✔

File: frontend/src/common/styles.js [hunk 1]
```diff
@@ -4,35 +4,40 @@ import Colors from './colors';
...
-    borderRadius: 10,
-    elevation: 2,
-    backgroundColor: '#10978b',
+    shadowColor: '#000',
+    shadowOpacity: 0.1,
+    shadowOffset: { x: 0, y: 4 },
+    shadowRadius: 6,
+    padding: 15,
+    marginBottom: 10,
+    backgroundColor: '#fff',
```
Comment (evasimonyi) [inline frontend/src/common/styles.js]: isn't this color in the common colors already?

```suggestion
    backgroundColor: Colors.whiteColor,
```

File: frontend/src/common/styles.js [hunk 2]
```diff
@@ -4,35 +4,40 @@ import Colors from './colors';
...
-    paddingTop: 12,
-    paddingHorizontal: 12,
-    paddingBottom: 13,
-    display: 'flex',
-    flexDirection: 'row',
-    alignItems: 'center',
-    borderRadius: 10,
-    elevation: 2,
-    backgroundColor: '#10978b',
+    shadowColor: '#000',
```
Comment (evasimonyi) [inline frontend/src/common/styles.js]: if we store the white color in the common colors, I think we can store the black too, opinion?

Comment (Kyya) [reply frontend/src/common/styles.js]: Yes, of sure

**Pull request merged**